app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -27,11 +27,16 @@ var AnalyticsAA = React.createClass({
     const done = ids.filter(v => v.inTheDatabase).length;
     const total = ids.length;
     const completion = ((done / total) * 100);
+    const bucket = 'https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/';
+    const file = data.provinceName.match(/ /g) ? data.provinceName.replace(' ', '+') : data.provinceName;
```

Comment:
these are the links shown in the s3 console
`https://s3.amazonaws.com/openroads-vn-dumps/by-province-name/Thai+Binh.geojson`
when I tried w/spaces the resource was not findable.